stream/src/public_key/connection.rs

```diff
@@ -31,6 +31,22 @@ pub struct Connection<Si: Sink, St: Stream> {
 }
 
 impl<Si: Sink, St: Stream> Connection<Si, St> {
+    pub fn new(
```

Comment:
I'd like to use this in benches only, similar to `#[cfg(test)]`, not sure if you have any ideas

Reply:
I don't think there is a way (at least not one I know of).

It may be ok to actually export it as something along the lines of `pre-established` (where someone has their own shared secret and doesn't want to use the "happy path" handshake) 🤔